lib/tasks/search.rake

```diff
@@ -16,6 +16,6 @@ namespace :search do
       exit
     end
 
-    SearchClient.indices.delete(index: "*")
```

Comment:
I'm wondering if we should actually keep this, its a great way to "reset" your cluster bc it will remove ALL the indexes, not just the ones we expect to find. But I can live with this for now and we can add it back if we need to.

Reply:
Now that you mention it might be an issue for devs who share an instance of ES with other apps on their machines, or would it namespaced somehow?

Reply:
Huh, that's a good point, I never thought about that. We could namespace our indexes but I really don't want to do an index rename at this point. I'm game to leave it like this. We are getting to a more solid state now with auto index creation turned off so the chances of this happening are a lot less.